### PR TITLE
chore(docs): README improvement and blog template

### PR DESCRIPTION
**Pull request by amyshwang**
1. Adding blog post template and guidelines
2. Editing the contribution section of README.md:
- Pointing to existing guidelines
- Adding Legal section
This is the first step to compile a `contribution.md` but for now I'd use the existing section to flesh out the content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,25 +58,45 @@ yarn serve
...
-pull requests.
+We are always happy to have contributions to the project whether it is
+documentation, bug reports, blog posts, or feedback. To get started with
+contributing:
 
-## Guidelines
+- Have a look through
+  [GitHub issues](https://github.com/questdb/questdb.io/issues).
+- Read this section for guidelines.
+- For blog posts, follow [blog guidelines](blog/__guidelines/blog.md)
```
Comment (amyshwang) [inline README.md:68]: The link should be added after the new file has been merged to `master`?

File: blog/__guidelines/blog_contribution.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+We welcome contributions to our blog posts. This is a way for you to contribute to the open source movement by writing technical content. We’re looking for content that features technical subjects to help more developers to start exploring and adopting QuestDB.
+
+This page describes the process and guidelines for submitting tutorial ideas and publishing content on our website.
+
+## What can you write about?
+
+We are looking for original content in generally these categories:
+
+- Articles that explore time series database landscape with a focus on QuestDB.
```
Comment (pswu11) [inline blog/__guidelines/blog_contribution.md:13]: Maybe, 

Articles that explore topics related to time series analysis, time series databases, and the use cases, ideally covering QuestDB or with a focus on QuestDB. 

For those SEO-related articles, they don't usually have a focus on QuestDB but is related. (see #seo channel on slack)

File: blog/__guidelines/blog_contribution.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Articles that explore time series database landscape with a focus on QuestDB. 
+- Step-by-step tutorials that walk the reader through creating a real-world project with QuestDB from start to finish. These can also cover topics related to using QuestDB with other relevant tools in different use cases.
+
+If there is a topic that you might find relevant for the QuestDB community, but doesn’t fall under the above categories, you can still submit your proposal and we’ll review it.
+
+## How does it work?
+
+1. Explore QuestDB. Join our [Slack channel](https://slack.questdb.io/) to discuss and ask questions about QuestDB.
+2. Raise an [GH issue](https://github.com/questdb/questdb.io/issues/new/choose) and describe your content idea.
```
Comment (pswu11) [inline blog/__guidelines/blog_contribution.md:21]: perhaps we can be specific about what we expect in the proposal, (for example, headline, abstract, target audience, etc.)

Comment (amyshwang) [reply blog/__guidelines/blog_contribution.md:21]: I would not want to repeat the information here if we define it well in the template but yes will point to the template.

Comment (amyshwang) [reply blog/__guidelines/blog_contribution.md]: @pswu11 I realised the template can be amended as part of the PR too so I have added a commit!

File: .github/ISSUE_TEMPLATE/submit-a-tutorial.md [hunk 4]
```diff
@@ -10,26 +10,31 @@ assignees: ""
...
 Use this section to give a description of what your tutorial is about.
 
-Examples:
+- Title:
+- Relevant industries:
+- Target audience:
 
-> I would like to write a tutorial that shows how to use QuestDB:
->
-> - "for storing IoT sensor data..."
```
Comment (pswu11) [inline .github/ISSUE_TEMPLATE/submit-a-tutorial.md:26]: It'd be nice to provide some examples 
like title (Building .... with QuestDB and xxx), who is this for (e.g. Data Engineer who is building a data pipeline for and IoT project / Data scients who focus on query xxx...), etc.

Comment (pswu11) [reply .github/ISSUE_TEMPLATE/submit-a-tutorial.md:26]: In that case, if someone is new they have a good reference on what makes sense.

File: .github/ISSUE_TEMPLATE/submit-a-tutorial.md [hunk 5]
```diff
@@ -10,26 +10,31 @@ assignees: ""
...
-> - "for creating data visualizations..."
-> - "with an ML library..."
-> - "for monitoring an application..."
+<!--
+Please select or describe the audience your proposal targets in terms of readers' familiarity with QuestDB
```
Comment (pswu11) [inline .github/ISSUE_TEMPLATE/submit-a-tutorial.md:26]: do you intend to show level of difficulty/level experience someone has with QuestDB here below?

Comment (amyshwang) [reply .github/ISSUE_TEMPLATE/submit-a-tutorial.md:26]: Yes this is explaining what L29-L32 means. Is this not clear enough?

File: .github/ISSUE_TEMPLATE/submit-a-tutorial.md (hunk 4 above)
Comment (amyshwang) [reply .github/ISSUE_TEMPLATE/submit-a-tutorial.md:26]: Added this to `blog_contribution.md`. I think that is a more suitable place for examples.

**Comment by pswu11** [comment]
@amyshwang I might have missed it somewhere, I like that we have templates - just where do potential contributors find the links to these templates? I only see guidelines for docs on README and there's no individual CONTRIBUTING.md in this repo.

**Comment by amyshwang** [comment]
@pswu11 Re templates - I added a whole section in L93 linking to all the templates currently available - do you mean that section is not clear to find?

**Comment by pswu11** [comment]
> @pswu11 Re templates - I added a whole section in L93 linking to all the templates currently available - do you mean that section is not clear to find?

OK I overlooked it, ignore the last comment. XD

**Review by pswu11 — APPROVED** [review]

**Pull request merged**